### PR TITLE
Infodev 3650

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by WilliamLoy** [comment]
@cyrichardson This one is ready for you :) Sorry that it is a novel!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ed6401afb5/how-to/)

**Comment by cyrichardson** [comment]
On it right after a meeting, @WilliamLoy.

**Comment by cyrichardson** [comment]
Suggestions here, @WilliamLoy:
[Microsoft-Skype-for-Business-FAQ_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2989658/Microsoft-Skype-for-Business-FAQ_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-6a96598c7b/how-to/)

File: content/skype-for-business/skype-for-business-faq.md [hunk 1]
```diff
@@ -38,21 +36,21 @@ If you're not getting sound, check that:
     **Options &gt; Personal.**
 2.  Uncheck the box labeled **Automatically start Skype for Business
     when I log on to Windows**. The next time you log on, if you want to
-    use Skype for Business you'll need to select it from the Start menu.
+    use Skype for Business you'll need to select it from the **Start** menu.
```
Comment (catlook) [inline content/skype-for-business/skype-for-business-faq.md]: ... Business, you'll ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dfe4a2cbae/how-to/).

**Pull request merged**